fabric8-services/fabric8-wit
# Migrate to `dep` for dependency management (OSIO#2873)

Generating the `Gopkg.*` by running `dep init`, then
adapting the `Makefile` by replacing all calls to `glide`
with calls to `dep`.

Also:
- download specified version of dep and use it
- update doc, remove assetfs, add .dockerignore
- set version on goa and satoro/go.uuid
- updatr to latest version of auth (to date)
- include fabric8-auth/account pkg in dependencies
- generate fabric8-auth/account/tenant in vendor
- update stretchr/testify to 1.2.1
- Add test-keeper.yml to validate build (the 'Ike'
  bot will no longer complain if there is no test
  in the PR while it only brings changes to the
  `Makefile`.)
- remove the `glide.*` files as they are
  no longer needed.

Fixes openshiftio/openshift.io#2873

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>
